web-client/src/modules/create/routes.tsx

```diff
@@ -11,13 +12,16 @@ const CreatePostContainer = lazy(() =>
 const CreatePostPage: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
   const { type } = useParams() as Record<string, string>;
-  return <CreatePostContainer type={type as CreatePostTypes} />;
+  return <CreatePostContainer createPostType={type as CreatePostTypes} />;
 };
 
 const Routes = (): ReactElement => (
   <Suspense fallback={<LoadingWrapper />}>
     <Switch>
-      <Route path={CreatePostLocation.path} component={CreatePostPage} />
+      <ProtectedRoute
```

Comment:
don't use protected route here, we need it to be partially protected and the whole module is anyway rendered with partially protected in masterpage